src/assets/RealAsset/RealAsset/realAsset.js

```diff
@@ -86,18 +86,29 @@ class RealAsset extends Asset {
       ownership: {
         get: () => this._ownership,
         set: (newOwnership) => {
+          if (newOwnership instanceof Array) {
            var sum=0;
-           var arrayKey=Object.keys(newOwnership)
-           for (var i = 0; i < arrayKey.length; i++)
+           var flag=false;
```

Comment:
I don't think you need the `flag` variable. When you throw an error, the code will stop executing. This means that at line 103 when you do `if (sum == 1 && flag == false)` the `flag` will always be false, because when you assign it to `true` at line 98, you immediately throw an error. So if the code execution reaches line 103, then `flag` is guaranteed to be false.